apps/native-component-list/src/screens/Location/GeofencingScreen.tsx

```diff
@@ -189,10 +186,13 @@ TaskManager.defineTask(GEOFENCING_TASK, async ({ data: { region } }: { data: any
   // tslint:disable-next-line no-console
   console.log(`${stateString} region ${region.identifier}`);
 
-  await Notifications.presentLocalNotificationAsync({
-    title: 'Expo Geofencing',
-    body: `You're ${stateString} a region ${region.identifier}`,
-    data: region,
+  await Notifications.scheduleNotificationAsync({
+    content: {
+      title: 'Expo Geofencing',
+      body: `You're ${stateString} a region ${region.identifier}`,
+      data: region,
+    },
+    trigger: null,
```

Comment:
Is it required to explicitly pass `null` as a trigger?

Reply:
cc @sjchmiela

Reply:
I've copied it from the documentation. So, I assumed that it's necessary to pass `null`.  But, I'm not sure now.

Reply:
It's necessary 😅 
![image](https://user-images.githubusercontent.com/9578601/83543152-47b54f80-a4fc-11ea-85f0-777d40c09151.png)

Reply:
UX designers are crying 😢